Sources/performance-worker/WorkerService.swift

```diff
@@ -98,3 +169,44 @@ final class WorkerService: Grpc_Testing_WorkerService.ServiceProtocol, Sendable
     throw RPCError(code: .unimplemented, message: "This RPC has not been implemented yet.")
   }
 }
+
+@available(macOS 13.0, iOS 16.0, watchOS 9.0, tvOS 16.0, *)
+extension WorkerService {
+  private func setupServer(_ config: Grpc_Testing_ServerConfig) async throws -> GRPCServer {
+    let server = GRPCServer(transports: [], services: [BenchmarkService()])
+    let initialServerState = try await State.ServerState(server: server)
```

Comment:
It's weird to pass the state into the state machine. The `ServerState` is an implementation detail of the `State` so this is a bit of layering violation. It's clearer to pass in whatever is needed to construct the server state, i.e. the server and the stats.